tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/VulnerabilityBatchTests.Bundle.cs

```diff
@@ -0,0 +1,273 @@
+using Datadog.Trace.Iast;
+using Datadog.Trace.Iast.SensitiveData;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.FluentAssertionsExtensions.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using FluentAssertions;
+using Xunit;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Datadog.Trace.Security.Unit.Tests.IAST.Tainted;
+
+/// <summary>
+/// VulnerabilityBatchTests for COMMAND_INJECTION
+/// </summary>
+public partial class VulnerabilityBatchTests
+{
+    private static JsonSerializerSettings _serializerSettings = VulnerabilityBatch.JsonRedactionSettings;
+    private static EvidenceRedactor _redactor = Utils.GetDefaultRedactor();
+
+    [Fact]
+    public void YamlTestBundle()
+    {
+        List<string> successfull = new List<string>();
+        List<string> failed = new List<string>();
+        string yamlContent = ResourceHelper.ReadAllText<VulnerabilityBatchTests>("evidence-redaction-suite.yml");
+
+        var deserializer = new DeserializerBuilder().WithNamingConvention(CamelCaseNamingConvention.Instance).Build();
+        var suite = deserializer.Deserialize<TestSuite>(yamlContent);
+
+        foreach (var test in suite.Suite)
+        {
+            var parametersList = CombineParameters(test.Parameters);
+            foreach (var parameters in parametersList)
+            {
+                var description = ReplaceParameters(test.Description, parameters);
+                var context = ReplaceParameters(test.Context, parameters);
+                var input = ReplaceParameters(test.Input, parameters);
+                var expected = ReplaceParameters(test.Expected, parameters);
+
+                try
+                {
+                    if (test.Type == "SOURCES")
+                    {
+                        ExecuteSourcesTest(description, context, input, expected, parameters);
+                    }
+                    else if (test.Type == "VULNERABILITIES")
+                    {
+                        ExecuteVulnerabilitiesTest(description, context, input, expected, parameters);
+                    }
+
+                    var txt = $"PASSED: {description}";
+                    successfull.Add(txt);
+                }
+                catch (System.Exception err)
+                {
+                    var txt = $"FAILED: {description} : {err}";
+                    failed.Add(txt);
+                }
+            }
+        }
+
+        // Produce the output
+        var separator = System.Environment.NewLine + "  ";
+        if (failed.Count > 0)
+        {
+            throw new System.Exception($"{failed.Count} out of {failed.Count + successfull.Count} sub test FAILED. {separator} {string.Join(separator, failed)}");
+        }
+        else
+        {
+            System.Console.WriteLine($"{successfull.Count} sub test PASSED. {separator} {string.Join(separator, successfull)}");
+        }
```

Comment:
You can't write to `Console` in xunit tests, so no point in this line

Reply:
Redirected to the TestOutputHelper in d2ebf75b7c559207788df84f837aecc7c71343e4